include/pisa/scorer/bm25.hpp

```diff
@@ -7,26 +7,34 @@
 #include "index_scorer.hpp"
 namespace pisa {
 
+// Implements the Okapi BM25 model. k1 and b are both free parameters which
+// alter the weight given to different aspects of the calculation.
+// We adopt the defaults recommended by the following resource - A. Trotman,
+// X-F. Jia, and M. Crane: "Towards an Efficient and Effective Search Engine,"
+// in Proceedings of the SIGIR 2012 Workshop on Open Source Information
+// Retrieval (OSIR), 2012.
```

Comment:
Let's use triple slash `///` to indicate it's a piece of documentation, and remove the empty line between the comment and the struct.

The reason is that if we ever decide to use a API doc generator, such as doxygen, it will be automatically usable.